RijsMuseum/app/src/main/java/com/example/rijsmuseum/fragments/ArtifactFragment.kt

```diff
@@ -0,0 +1,67 @@
+package com.example.rijsmuseum.fragments
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProvider
+import com.example.network.models.Collections
+import com.example.rijsmuseum.R
+import com.example.rijsmuseum.adapter.RecyclerAdapter
+import com.example.rijsmuseum.databinding.FragmentArtifactBinding
+import com.example.rijsmuseum.viewmodel.ArtifactViewModel
+
+class ArtifactFragment : Fragment(), RecyclerAdapter.OnItemClickListener {
```

Comment:
why would you name it ArtifactFragment when it shows a list of artefacts? Missing "s" may be? Should be plural